apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/AddActionModal.tsx

```diff
@@ -0,0 +1,463 @@
+"use client";
+
+import { createActionClassAction } from "@/app/(app)/environments/[environmentId]/actions/actions";
```

Comment:
please don't cross reference actions from other routes. Routes should be independent from another so that we can just move or remove routes without affecting other routes. We should add a separate action to the [surveyId]/edit route.